qmail-remote.c

```diff
@@ -118,36 +131,50 @@ substdio smtpfrom = SUBSTDIO_FDBUF(saferead,-1,smtpfrombuf,sizeof(smtpfrombuf));
 
 stralloc smtptext = {0};
 
-static void get(unsigned char *uc)
```

Comment:
Haven't checked, just guessing we should probably retain the old name as a synonym in case patches call it.